libs/blocks/section-metadata/section-metadata.css

```diff
@@ -17,6 +17,11 @@
   color: rgb(34 34 34);
 }
 
+.section.hide-sticky-section,
+.section.close-sticky-section {
```

Comment:
@overmyheadandbody The promo sections can have a close button as per the proposed designs and in that case the observer would need to be removed. When the promobar element close action is triggerd it can add the class close-sticky-section to the section. This would hide the section and the intersection observer would disable the observers if the class is applied to the section.
hide sticky section would not enforce the observers to unobserve and the class would only toggle in case of scrolling.